octobot_trading/exchanges/exchange_manager.py

```diff
@@ -253,21 +258,30 @@ def _load_config_symbols_and_time_frames(self):
             self._logger.error("Failed to load client from REST exchange")
             self._raise_exchange_load_error()
 
+    async def _initialize_simulator_time_frames(self):
+        for importer in self.exchange.exchange_importers:
+            for symbol, time_frames in get_available_time_frames_by_symbol(importer).items():
+                self.client_time_frames[symbol] = [time_frame.value for time_frame in time_frames]
+
     # SYMBOLS
     def symbol_exists(self, symbol):
         if self.client_symbols is None:
             self._logger.error(f"Failed to load available symbols from REST exchange, impossible to check if "
-                              f"{symbol} exists on {self.exchange.name}")
+                               f"{symbol} exists on {self.exchange.name}")
             return False
         return symbol in self.client_symbols
 
     # TIME FRAMES
     def time_frame_exists(self, time_frame, symbol=None):
         if CONFIG_WILDCARD in self.client_time_frames and not self.client_time_frames[CONFIG_WILDCARD]:
             return False
-        elif CONFIG_WILDCARD in self.client_time_frames or symbol is None:
+        elif CONFIG_WILDCARD in self.client_time_frames and symbol is None:
             return time_frame in self.client_time_frames[CONFIG_WILDCARD]
 
+        elif symbol is None:
```

Comment:
Why do you need a specific check when symbol is None ?

Reply:
Because for now symbol is always None given the current usage of this method (only the time_frame argument is set)